### PR TITLE
updated test to include property column count, updated model query.

### PR DESCRIPTION
**Pull request by ECJMac**
Updated test to include property column count.
Updated model to join reviews and comments tables and add column column count to response.

### CODE REVIEW

**Review by Angaca — CHANGES_REQUESTED** [review]
You want to expand your test a bit more testing where there are more than 1 comment or 0 comments as your query is not accurate enough, COUNT(*) is too generic, you want to count more specifically inside the comments table

**Comment by ECJMac** [comment]
Updated test file reviews comment_count is a number
Updated model query to target comment_id for comment count
Updated model query to return comment_count as a number not a string

File: __tests__/app.test.js [hunk 1]
```diff
@@ -36,18 +36,34 @@ describe("app tests", () => {
...
+            expect(typeof reviews.comment_count).toBe("number");
+          });
+      });
+      test("200: comment count works for multiple comments", () => {
+        const REVIEW_ID = 2;
+        return request(app)
+          .get(`/api/reviews/${REVIEW_ID}`)
+          .expect(200)
+          .then(({ body: { reviews } }) => {
+            console.log(reviews);
```
Comment (HeatherEver) [inline __tests__/app.test.js]: delete console logs once used please :)

File: db/models/models.js [hunk 2]
```diff
@@ -19,3 +22,8 @@ exports.selectUsers = () => {
     return users.rows;
   });
 };
+
+// SELECT reviews.*, COUNT(*) AS comment_count FROM reviews
+//     LEFT JOIN comments ON reviews.review_id = comments.review_id
+//     WHERE reviews.review_id = 2
+//     GROUP BY reviews.review_id;
```
Comment (HeatherEver) [inline db/models/models.js]: delete any code that you're not using before you commit :)

File: __tests__/app.test.js [hunk 3]
```diff
@@ -36,18 +36,34 @@ describe("app tests", () => {
...
-              designer: "Uwe Rosenberg",
-              owner: "mallionaire",
-              review_img_url:
-                "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
-              review_body: "Farmyard fun!",
-              category: "euro game",
-              created_at: "2021-01-18T10:00:20.514Z",
-              votes: 1,
-            });
+            console.log(reviews);
```
Comment (HeatherEver) [inline __tests__/app.test.js]: a console log to delete please :)

**Review by HeatherEver — COMMENTED** [review]
Nice one! Just a couple of bits to refactor

**Pull request merged**